<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x><y>0</y><width>980</width><height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grabación de caminata</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
    <item>
     <widget class="QFrame" name="frameVideo">
      <property name="frameShape"><enum>QFrame::NoFrame</enum></property>
      <layout class="QVBoxLayout" name="verticalLayoutVideo">
       <item>
        <widget class="QLabel" name="videoLabel">
         <property name="minimumSize">
          <size><width>640</width><height>360</height></size>
         </property>
         <property name="alignment"><set>Qt::AlignCenter</set></property>
         <property name="text">
          <string>Vista de cámara</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #111; color: #777; border-radius: 12px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameSide">
      <property name="styleSheet">
       <string notr="true">
#card { background: #1c1f24; border-radius: 14px; }
QLineEdit, QPushButton, QProgressBar { border-radius: 8px; }
QLineEdit { background:#0f1115; border:1px solid #2a2f37; color:#e6e6e6; padding:8px;}
QPushButton { background:#2a6df4; color:white; padding:10px; border:none; font-weight:600;}
QPushButton:disabled { background:#3a3f47; color:#9aa0a6;}
QLabel#title { font-size:18pt; font-weight:700; color:#ffffff;}
QLabel#subtitle { font-size:10pt; color:#9aa0a6;}
QLabel#status { font-size:11pt; }
QLabel#countdown { font-size:36pt; font-weight:700; color:#ffffff; }
QProgressBar { background:#0f1115; border:1px solid #2a2f37; color:white; text-align:center; height:16px;}
QProgressBar::chunk { background:#2a6df4; border-radius:8px;}
       </string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutSide" stretch="0,0,0,0,0,1,0,0">
       <item>
        <widget class="QFrame" name="cardHeader">
         <property name="objectName"><string>card</string></property>
         <layout class="QVBoxLayout" name="layoutHeader">
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="objectName"><string>title</string></property>
            <property name="text"><string>Grabación de caminata</string></property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="subtitleLabel">
            <property name="objectName"><string>subtitle</string></property>
            <property name="text"><string>Dos tomas: horizontal y frontal</string></property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="cardUser">
         <property name="objectName"><string>card</string></property>
         <layout class="QVBoxLayout" name="layoutUser">
          <item>
           <widget class="QLabel" name="nameLbl">
            <property name="text"><string>Nombre de usuario</string></property>
            <property name="styleSheet"><string notr="true">color:#c9d1d9;</string></property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameEdit">
            <property name="placeholderText"><string>ej. juanperez</string></property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="cardControls">
         <property name="objectName"><string>card</string></property>
         <layout class="QVBoxLayout" name="layoutControls">
          <item>
           <widget class="QPushButton" name="startBtn">
            <property name="text"><string>Iniciar proceso</string></property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="goBtn">
            <property name="text"><string>Comenzar cuenta regresiva</string></property>
            <property name="enabled"><bool>false</bool></property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="instructionLbl">
            <property name="text"><string>Instrucciones aparecerán aquí…</string></property>
            <property name="styleSheet"><string notr="true">color:#e6e6e6;</string></property>
            <property name="wordWrap"><bool>true</bool></property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="countdownLbl">
            <property name="objectName"><string>countdown</string></property>
            <property name="text"><string/></property>
            <property name="alignment"><set>Qt::AlignHCenter|Qt::AlignVCenter</set></property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="cardStatus">
         <property name="objectName"><string>card</string></property>
         <layout class="QVBoxLayout" name="layoutStatus">
          <item>
           <widget class="QLabel" name="statusLbl">
            <property name="objectName"><string>status</string></property>
            <property name="text"><string>Estado: Esperando</string></property>
            <property name="styleSheet"><string notr="true">color:#6aa6ff;</string></property>
            <property name="wordWrap"><bool>true</bool></property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value"><number>0</number></property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="spacer"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeHint"><size><width>20</width><height>40</height></size></property></spacer>
       </item>
       <item>
        <widget class="QLabel" name="footerLbl">
         <property name="text"><string></string></property>
         <property name="alignment"><set>Qt::AlignCenter</set></property>
         <property name="styleSheet"><string notr="true">color:#5f6a7a;</string></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
